docs/resources/faq.md

```diff
@@ -147,7 +147,7 @@
 
 #### Does yearn hire or pay contributors?
 
-- [Contributors](https://docs.yearn.finance/contributing/contribute#how-to-contribute)
+- [Contributors](http://docs.yearn.finance/contributing/operations/decision-making#a-personal-opportunity)
```

Comment:
this is a different topic about decision making. This is a better link: https://yearnfinance.notion.site/Join-Us-3e9c95b9bd7846a18c0f1cbe6ab05eda

Reply:
Included that page as it talks about compensation. Updated to the better link in 827f413b4bd1d45e099619eb21ce511d5fbe740e